net.sourceforge.squirrel_sql.appdata.xml

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Ideally should be sent and maintained upstream.

Reply:
Merge request:
https://sourceforge.net/p/squirrel-sql/git/merge-requests/75/